Makefile

```diff
@@ -19,7 +19,7 @@
 
 # values used in workspace-status.sh
 DOCKER_REGISTRY?=cockroachdb
-DOCKER_IMAGE_REPOSITORY?=cockroachdb-operator
+DOCKER_IMAGE_REPOSITORY?=cockroach-operator
```

Comment:
@rail this was WRONG.  Is this used in the release process?

Reply:
The release script sets it to [cockroachdb/cockroach-operator](https://github.com/cockroachdb/cockroach-operator/blob/a65d8e9dcb100035010bbd8ec294529bb29837a7/build/release/teamcity-publish-release.sh#L35)

Reply:
Yah we should have that in the Makefile, I don’t like stuff spread all over the place 😂